R/pkg/R/utils.R

```diff
@@ -930,3 +930,7 @@ getOne <- function(x, envir, inherits = TRUE, ifnotfound = NULL) {
 traverseParentDirs <- function(x, count) {
   if (dirname(x) == x || count <= 0) x else c(x, Recall(dirname(x), count - 1))
 }
+
+is_arrow_conf_set <- function() {
```

Comment:
Let's don't do this here In PySpark and Scala side, they have a standardised way and place to do it for configurations but R doesn't have. It would need a separate look and we should think where/how we will organise this.

Reply:
OK reverted